src/main/java/server/ProxyServer.java

```diff
@@ -36,23 +39,71 @@
  */
 public class ProxyServer extends ProxyServiceImplBase {
     private static final Logger logger = LoggerFactory.getLogger(ProxyServer.class);
-
     private Member member;
+    private Config config;
 
-    public ProxyServer() throws IOException {
+    ProxyServer() throws IOException {
+        config = ConfigFactory.load();
         TokenIO lib = initializeSdk();
         member = initializeMember(lib);
     }
 
-    public TokenIO initializeSdk() throws IOException {
-        Path keys = Files.createDirectories(Paths.get("./keys"));
+    @Override
+    public void getMember(GetMemberRequest request,
+                             StreamObserver<GetMemberResponse> responseObserver) {
+        execute(responseObserver, () -> {
+            logger.info("Get({})", TextFormat.shortDebugString(request));
```

Comment:
Can you just install the logging interceptor that we have? And metrics? Then they have all the logs in the format that we use and we when we ask them for the logs they look exactly as ours.

Reply:
Ok